PyInstaller/utils/hooks/__init__.py

```diff
@@ -65,29 +65,36 @@ def __exec_python_cmd(cmd, env=None):
         pp = os.pathsep.join([pp_env.get('PYTHONPATH'), pp])
     pp_env['PYTHONPATH'] = pp
 
-    txt = exec_python(*cmd, env=pp_env)
-    return txt.strip()
+    if ret_stdout:
+        txt = exec_python(*cmd, env=pp_env)
+        return txt.strip()
+    else:
+        return exec_python_rc(*cmd, env=pp_env)
+
+
+def __exec_statement(statement, ret_stdout=True):
+    statement = textwrap.dedent(statement)
+    cmd = ['-c', statement]
+    return __exec_python_cmd(cmd, ret_stdout=ret_stdout)
 
 
 def exec_statement(statement):
     """
     Executes a Python statement in an externally spawned interpreter, and
     returns anything that was emitted in the standard output as a single string.
     """
-    statement = textwrap.dedent(statement)
-    cmd = ['-c', statement]
-    return __exec_python_cmd(cmd)
+    return __exec_statement(statement, ret_stdout=True)
 
 
-def exec_script(script_filename, *args, env=None):
+def exec_statement_rc(statement):
     """
-    Executes a Python script in an externally spawned interpreter, and
-    returns anything that was emitted in the standard output as a
-    single string.
-
-    To prevent misuse, the script passed to utils.hooks.exec_script
-    must be located in the `PyInstaller/utils/hooks/subproc` directory.
```

Comment:
This is a restriction of the new `__exec_script`, thus is should be kept in tht doc-string.

Reply:
OK, will add/modify the doc-string for the new `__exec_script` so that limitation is documented both in the public-facing and the private function.